src/components/Arrow/Arrow.tsx

```diff
@@ -24,14 +26,28 @@ const Arrow: FC<ArrowProps> = ({ edge, edgeName, widthOffset, currentNode }) =>
   const edgeNameNoWhitespace = edgeName.replace(' ', '');
   const arrowheadId = `arrowhead-${edgeNameNoWhitespace}`;
 
+  const CustomTooltip = withStyles({
+    tooltip: {
+      fontSize: 14
+    }
+  })(Tooltip);
+
```

Comment:
Sorry I didn't post this yesterday, was waiting for Noranda to confirm - we should add these to the MuiOverrides in `theme.tsx` since we will probably want this same styling for all tooltips